frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -67,40 +67,40 @@ function BookInfo:show(file, book_props)
     table.insert(kv_pairs, { _("Folder:"), BD.dirpath(filemanagerutil.abbreviate(directory)), separator = true })
 
     -- book_props may be provided if caller already has them available
-    -- but it may lack 'pages', that we may get from sidecar file
+    -- but it may lack "pages", that we may get from sidecar file
```

Comment:
if you're going to edit comments, I'd prefer if you make them proper sentences (uppercase first letter at the start, period at the end). ;-)

Reply:
That one was an S&R pass ;p.